README.md

```diff
@@ -24,6 +24,8 @@ The HMDA Frontend monorepo hosts the public facing applications for the collecti
         * [Configure the UI](#configure-the-ui)
       - [Updating or Previewing Documentation](#updating-or-previewing-documentation)
     + [Running via Docker](#running-via-docker)
+    + Creating and editing Markdown
+      - In order to correctly display a Table of Contents, headings in Markdown files need to conform to certain standards. Please see [standarized documentation process](https://github.com/cfpb/hmda-frontend/wiki/Standardized-process-for-generating-or-updating-markdown-files) for details.
```

Comment:
Tabular version: https://github.com/cfpb/hmda-frontend/issues/1565#issuecomment-1304325354

Reply:
Tabular version is a big improvement to the doc.

> Option 2) Link "Creating and editing Markdown" directly to the wiki page and use the the 'In order to correctly...' text there.

I think this would be the better option.